Gemfile.lock

```diff
@@ -361,6 +390,7 @@ DEPENDENCIES
   rails!
   rake (>= 10.3)
   redcarpet (~> 3.2.3)
+  redis (~> 3.0)
```

Comment:
Cool. I think whatever strategy we apply for `redis` should also be used for `eventmachine` as it's pretty heavyweight as well. We should see if we can drop the dependency on `coffee-rails` as well, as that's not something that `rails` currently has a dependency.